src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md

```diff
@@ -221,63 +181,26 @@ export default function() {
 
 </div>
 
-## Using checks
-
-Maybe we want to verify that an HTTP transaction worked and that the transaction time stayed below some acceptable value. We can use the [check()](/javascript-api/k6/check-val-sets-tags) function for this:
-
-<div class="code-group" data-props='{"labels": ["stages.js"], "lineNumbers": [true]}'>
+## Running cloud tests
 
-```javascript
-import http from 'k6/http';
-import { check, sleep } from 'k6';
-
-export let options = {
-  vus: 10,
-  duration: '30s',
-};
-
-export default function() {
-  let res = http.get('http://test.k6.io');
-  check(res, {
-    'status was 200': r => r.status == 200,
-    'transaction time OK': r => r.timings.duration < 200,
-  });
-  sleep(1);
-}
-```
+One of the goals with k6 is to support three execution modes to run your k6 tests:
 
-</div>
+- [Local](#running-local-tests): on your local machine or a CI server.
+- [Cloud](/cloud): on cloud infrastructure managed by `k6 Cloud`.
+- Clustered: on more than one machine managed by you. Not supported yet.
 
-_Using check() to verify that transactions work and are fast enough_
+The goal of k6 is to support running a test on the three execution modes without making modifications to the script. 
```

Comment:
This sentence just feels like a repeat of the sentence from above:
> One of the goals with k6 is to support three execution modes to run your k6 tests:

Maybe remove this sentence and add ", without making modifications to the script" to the sentence above the three bullet points?

Reply:
https://github.com/loadimpact/k6-docs/pull/26/commits/63afdcc5f86e883180b9cb050c595edea88c4720